### PR TITLE
Use karma to load jasmine specs.

### PR DESCRIPTION
**Pull request by blainesch**
The current issue here is that the browser can't find `jasmine`. The
reason is because jasmine assumes we are in node and doesn't put itself
on the `window` object.

This is an alternative WIP to #135 and #136

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +26,8 @@
     "build": "gulp build",
     "release": "gulp release --env=production",
     "start": "gulp start",
-    "test": "gulp start --env=test",
+    "pretest": "npm run build",
+    "test": "karma start",
```
Comment (blainesch) [inline package.json]: This should be converted to a gulp task.

**Comment by szwacz** [comment]
Would be awesome if it can work this way! 
The only problem is that ES6 modules are used in this project, so the tests can't be run without modules transpiling.

Will fiddle with it as well.

**Comment by blainesch** [comment]
@szwacz yes, if I can get this to work getting the transpilation working should be easy... just make the gulp task run build before it calls out to karma.

**Comment by szwacz** [comment]
Maybe drop jasmine if there is so many problems with it? https://github.com/jprichardson/electron-mocha

**Comment by szwacz** [comment]
Hoa! Was able to make it running in 15 minutes. Yes, I think this is the way to go. @blainesch would you like to continue in that direction? It looks 10 times easier to achieve.

**Comment by szwacz** [comment]
I've created branch https://github.com/szwacz/electron-boilerplate/tree/mocha for work on it. And will return fiddling with it on evening. If you still like to contribute (I hope you do :)) please send PRs to that branch.

**Pull request closed without merging**